secure_random/Makefile

```diff
@@ -0,0 +1,19 @@
+LIBSODIUM_VER_TAG=1.0.16
+
+TF_CFLAGS=$(shell python -c 'import tensorflow as tf; print(" ".join(tf.sysconfig.get_compile_flags()))' 2>/dev/null)
+TF_LFLAGS=$(shell python -c 'import tensorflow as tf; print(" ".join(tf.sysconfig.get_link_flags()))' 2>/dev/null)
+
+all: build-op
+
+build-lib-sodium:
+	# TODO CHECK IF SODIUM LIB EXISTS FIRST
+	git clone https://github.com/jedisct1/libsodium.git
```

Comment:
Instead of cloning the entire repository, we can download the version from GitHub as listed on their releases page: https://github.com/jedisct1/libsodium/releases

Reply:
We should probably put this into a `build/` folder that is git ignored. I'd also suggest that these make commands be a part of the larger Makefile as it's going to all be integrated together as a part of the same build pipeline.